Active Learning segment for Bond Length

Mayank447 opened the pull request:
- Added code to parse and extract molecular data and store them as pickle objects. 
 - Also did data analysis on bond length data. 
 - In the next iteration will add ML models to predict bond length.

Mayank447 committed:
Added data analysis code for Bond Lengths

Signed-off-by: Mayank Goel <56905718+Mayank447@users.noreply.github.com>

ghutchis commented:
I don't think the pickle archives are here. I know you were storing them on Dropbox - maybe add a script to download them?

ghutchis commented:
I'd also suggest some effort to tune the -0.25 exponent (through botorch? 😄) by minimizing MAE or RMSE.

Mayank447 committed:
Add files via upload

Signed-off-by: Mayank Goel <56905718+Mayank447@users.noreply.github.com>

Mayank447 committed:
Bond Length Prediction ML models and pickle files

Built baseline ML models for Active Learning and extracted Quantum chemical data using xtb-python

Signed-off-by: Mayank Goel <56905718+Mayank447@users.noreply.github.com>

Mayank447 commented:
Uploaded the pickle file to Github as well as added an external link.
All the work until building baseline ML models has been uploaded.

Mayank447 committed:
Active Learning for Bond Lengths

Optimized Bond Lengths using BayesOpt and Started with Bond Angles

Signed-off-by: Mayank Goel <56905718+Mayank447@users.noreply.github.com>

Mayank447 committed:
Baseline Bond Angle Prediction

Signed-off-by: Mayank Goel <56905718+Mayank447@users.noreply.github.com>

Mayank447 committed:
Active Learning upto Bond Angles

Signed-off-by: Mayank Goel <56905718+Mayank447@users.noreply.github.com>
